docs/providers.mdx

```diff
@@ -0,0 +1,37 @@
+---
+title: Providers
+icon: book-sparkles
+---
+
+Connect pulls data from `Providers`. `Providers` utilize public REST APIs and/or websockets from either centralized or decentralized exchanges. Below is a list of all available `Providers` in Connect.
```

Comment:
we use JSON RPC for some of the defi but idk if this is important

Reply:
[we use RPCs too](https://github.com/skip-mev/slinky/pull/604/commits/eabf32ecd845bc43a2e91dc3e583d5a81a533eef)